src/main/cpp/BUILD

```diff
@@ -132,6 +132,8 @@ cc_library(
         "//src/conditions:windows": [
             "/DBAZEL_SYSTEM_BAZELRC_PATH#\\\"%%ProgramData%%/bazel.bazelrc\\\"",
         ],
+        # For unix platforms, this can include environment variables in the
```

Comment:
Is it true that we don't to variable substitution on Windows? It seems like we do simply because %ProgramData% is expanded. And there is a call to ExpandEnvironmentStrings in path_windows.cc

Meta question, more for Serge.  Do we want to say "unix" or "*nix" or "most platforms".

Reply:
It's not true, I suppose I can write more words here about the Windows code

Reply:
Ah, there is already a comment above this for Windows, you just can't see it in the code review because it's too far away. I'll add more words to it anyway.

Reply:
Also spelled it "posix" in the end, to match the code.